android/build.gradle

```diff
@@ -26,6 +26,11 @@ def getExtOrIntegerDefault(name) {
 }
 
 android {
+  def agpVersion = com.android.Version.ANDROID_GRADLE_PLUGIN_VERSION
+  if (agpVersion.tokenize('.')[0].toInteger() >= 7) {
+      namespace "com.reactnativeshadowview"
+  }
```

Comment:
what is this code for?

Reply:
Just checking the Android Grade Plugin version to only define the namespace when it is supported by grade, so you can build with Gradle plugin Versions 5.X